2a_model/src/models/2_multitask_dense/Snakefile

```diff
@@ -17,7 +17,8 @@ rule all:
                  rep=list(range(config['num_replicates'])),
                  site_id=['01480870'],
                  year=[2012]),
-          f"{out_dir}/{config['exp_name']}_func_perf.csv"
+          f"{out_dir}/{config['exp_name']}_func_perf.csv",
+          f"{out_dir}/observed_func_perf.csv"
```

Comment:
I switched the "observed_func_performance" to this Snakefile b/c it only gets used once. The one in 0_baseline_LSTM also gets used by the 1_multitask models so the "observed_func_performance" was getting produced multiple times unnecessarily.